stopcovid/dialog/engine.py

```diff
@@ -356,3 +357,22 @@ def execute(self, dialog_state: DialogState) -> List[DialogEvent]:
                 phone_number=self.phone_number, user_profile=dialog_state.user_profile, sms=self.sms
             )
         ]
+
+
+class CreateUser(Command):
+    def __init__(self, phone_number: str, name: str, unit_id: int, employer_id: int):
+        super().__init__(phone_number)
+        self.name = name
+        self.unit_id = unit_id
+        self.employer_id = employer_id
+
+    def execute(self, dialog_state: DialogState) -> List[DialogEvent]:
+        return [
+            UserCreated(
+                phone_number=self.phone_number,
+                user_profile=dialog_state.user_profile,
+                name=self.name,
+                unit_id=self.unit_id,
+                employer_id=self.employer_id,
+            )
+        ]
```

Comment:
what is the purpose of the UserCreated event?

Reply:
i'm not sure I understand the question

Reply:
like what is this event used for? I understand that CreateUser causes the dialog engine to create a user. What does scadmin do when it gets the UserCreated event?

Reply:
I don't think scadmin needs to do anything with UserCreated--we just need to emit a `DialogEvent` to actually change anything in the user's `dialog_state`--i.e. `dialog_state` mutations happen in the `apply_to` method of the `DialogEvent`

Reply:
gotcha - yeah I think what I was sort of getting at is I was wondering if we need this event at all. But obvs doesn't hurt to have it!

Reply:
oh oh I get it - I see what you're saying now. The way the DE currently works we need to emit an event in order for dialog_state to get updated. So we need this event even though it's not used for anything

Reply:
yeah, exactly!